student_class.rb

```diff
@@ -2,11 +2,16 @@
 
 class Student < Person
   def initialize(classroom)
-    super
+    super()
     @classroom = classroom
   end
```

Comment:
- [ ] Kindly make the `classroom` attribute have a getter method. `attr_reader :classroom`. 👍